src/engine/shared/config_variables.h

```diff
@@ -64,6 +64,7 @@ MACRO_CONFIG_INT(InfMinPlayerNumberForMapVote, inf_min_player_number_map_vote, 1
 MACRO_CONFIG_INT(InfConWaitingTime, inf_con_waiting_time, 1, 0, 60, CFGFLAG_SERVER, "Number of seconds to wait before enter the game")
 MACRO_CONFIG_INT(InfCaptcha, inf_captcha, 0, 0, 1, CFGFLAG_SERVER, "Enable captcha")
 MACRO_CONFIG_INT(InfShockwaveAffectHumans, inf_shock_wave_affect_humans, 1, 0, 1, CFGFLAG_SERVER, "Explosion shockwave affect humans")
+MACRO_CONFIG_INT(InfBonusZoneTime, inf_bonus_zone_time, 60, 0, 10000, CFGFLAG_SERVER, "How long a human needs to stay in the bonus zone to get the bonus (in seconds)")
```

Comment:
i think this will make things very confusing. On all maps it is said that duration is 1 minute. Personally, i will never ever change this value
![image](https://user-images.githubusercontent.com/1087672/68999145-a7c80980-08bc-11ea-8310-fc9476091349.png)